src/components/play/layer-settings/LayerSettings.js.old

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react'
```

Comment:
huh, keep on seeing this file added 🤔 , maybe it's another .old file?

Reply:
please just ignore these for now, I'll get rid of them very soon.